server/views/pages/paginatedPersonSearchResults.njk

```diff
@@ -24,56 +23,73 @@
 
     <p class="govuk-body-l">
         {% if crn %}
-        <p class="govuk-body-l govuk-!-margin-bottom-3">
-            <span class='govuk-!-padding-right-2 govuk-!-font-weight-bold'>Case reference number (CRN):</span>
-             <span>{{ crn }}</span>
-        </p>
-        <a href="/search-by-crn" class="govuk-body-l govuk-link">Change <span class="govuk-visually-hidden">search term</span></a>
+            <p class="govuk-body-l govuk-!-margin-bottom-3">
+                <span class='govuk-!-padding-right-2 govuk-!-font-weight-bold'>Case reference number (CRN):</span>
+                <span>{{ crn }}</span>
+            </p>
+            {% if user.hasPpcsRole %}
```

Comment:
What about passing this in from the earlier page as a query parameter saying "the search page I have come from". That means we don't have to revisit this page if we ever create a new search page.  It also means that we don't need to get involved in checking role membership.

Reply:
Nice idea! Done!